Sources/DiscoveryV2/Models/ProjectDetails.swift

```diff
@@ -22,12 +22,17 @@ import Foundation
 public struct ProjectDetails: Codable, Equatable {
 
     /**
-     The project type of this project.
+     The type of project.
+     The `content_intelligence` type is a *Document Retrieval for Contracts* project and the `other` type is a *Custom*
+     project.
+     The `content_mining` and `content_intelligence` types are available with Premium plan managed deployments and
+     installed deployments only.
      */
     public enum TypeEnum: String {
         case documentRetrieval = "document_retrieval"
-        case answerRetrieval = "answer_retrieval"
+        case conversationalSearch = "conversational_search"
```

Comment:
Just confirming that we are OK with changing `answer_retrieval` to `conversational_search` here.

Reply:
Yes, answer_retrieval is no longer a valid option